charts/crowdsec/values.yaml

```diff
@@ -198,6 +198,16 @@ lapi:
 
 # agent will deploy pod on every node as daemonSet to read wanted pods logs
 agent:
```

Comment:
you can add a documentation comment above `additionalAcquisition` with a link to the datasources. Somethink like :

```
# -- To add custom acquisitions using available datasources (https://docs.crowdsec.net/docs/next/data_sources/intro)
```